src/Qir/Common/Include/SimulatorStub.hpp

```diff
@@ -15,113 +15,113 @@ namespace Quantum
         {
             throw std::logic_error("not_implemented: R");
         }
-        void Exp(long numTargets, PauliId paulis[], Qubit targets[], double theta) override
+        void Exp(long /* numTargets */, PauliId* /* paulis */, Qubit* /* targets */, double /* theta */) override
         {
             throw std::logic_error("not_implemented: Exp");
         }
-        void ControlledX(long numControls, Qubit controls[], Qubit target) override
+        void ControlledX(long /*numControls*/, Qubit* /*controls*/, Qubit /*target*/) override
         {
             throw std::logic_error("not_implemented: ControlledX");
         }
-        void ControlledY(long numControls, Qubit controls[], Qubit target) override
+        void ControlledY(long /*numControls*/, Qubit* /*controls*/, Qubit /*target*/) override
         {
             throw std::logic_error("not_implemented: ControlledY");
         }
-        void ControlledZ(long numControls, Qubit controls[], Qubit target) override
+        void ControlledZ(long /*numControls*/, Qubit* /*controls*/, Qubit /*target*/) override
         {
             throw std::logic_error("not_implemented: ControlledZ");
         }
-        void ControlledH(long numControls, Qubit controls[], Qubit target) override
+        void ControlledH(long /*numControls*/, Qubit* /*controls*/, Qubit /*target*/) override
         {
             throw std::logic_error("not_implemented: ControlledH");
         }
-        void ControlledS(long numControls, Qubit controls[], Qubit target) override
+        void ControlledS(long /*numControls*/, Qubit* /*controls*/, Qubit /*target*/) override
         {
             throw std::logic_error("not_implemented: ControlledS");
         }
-        void ControlledT(long numControls, Qubit controls[], Qubit target) override
+        void ControlledT(long /*numControls*/, Qubit* /*controls*/, Qubit /*target*/) override
         {
             throw std::logic_error("not_implemented: ControlledT");
         }
-        void ControlledR(long numControls, Qubit controls[], PauliId axis, Qubit target, double theta) override
+        void ControlledR(long /*numControls*/, Qubit* /*controls*/, PauliId /*axis*/, Qubit /*target*/, double /*theta*/) override
         {
             throw std::logic_error("not_implemented: ControlledR");
         }
         void ControlledExp(
-            long numControls,
-            Qubit controls[],
-            long numTargets,
-            PauliId paulis[],
-            Qubit targets[],
-            double theta) override
+            long /*numControls*/,
+            Qubit* /*controls*/,
+            long /*numTargets*/,
+            PauliId* /*paulis*/,
+            Qubit* /*targets*/,
+            double /*theta*/) override
         {
             throw std::logic_error("not_implemented: ControlledExp");
         }
-        void AdjointS(Qubit target) override
+        void AdjointS(Qubit) override
```

Comment:
For most of these the parameter name was commented out, but for `AdjointS` here and a few other functions throughout this file it was simply removed. I don't have a strong preference to either style, but do like following one pattern. For what it's worth, leaving the parameter names is helpful for understanding their intended use while on the other hand removing it entirely avoids the overhead of keeping them up to date if the names change in the future.

Reply:
Made them all commented out.